core/lib/types/src/contract_verification/api.rs

```diff
@@ -147,6 +147,19 @@ pub struct VerificationIncomingRequest {
     pub is_system: bool,
     #[serde(default)]
     pub force_evmla: bool,
+    #[serde(flatten)]
+    pub evm_specific: VerificationEvmSettings,
+}
+
+/// Settings for EVM verification, used only if
+/// `SourceCodeData` is `SolSingleFile`.
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct VerificationEvmSettings {
+    #[serde(default)]
+    pub evm_version: Option<String>,
```

Comment:
Isn't EVM version [supported by `zksolc`](https://github.com/matter-labs/era-compiler-solidity/blob/85bb573aadcf3f0772d6f898ceaa7b9487f12e0c/docs/src/03-standard-json.md#input-json)? Or by `VerificationEvmSettings` you don't necessarily mean settings related to EVM bytecode toolchains?

Reply:
It is supported, but AFAIU it's rather meaningless, since we're not compiling to any particular bytecode. Not sure if it will make sense to try using it outside of the EVM context (especially given that it's not used in the block explorer FE).